.github/workflows/test.yml

```diff
@@ -50,10 +50,18 @@ jobs:
 
     steps:
       - name: Install prerequisites
+        env:
+          S3CONFIG: ${{ secrets.S3CONFIG }}
         run: |
           sudo apt update
-          sudo apt install -y task-spooler time
+          sudo apt install -y task-spooler time s3cmd unzip
           tsp -S 48
+          mkdir -p $HOME/.s3cfg
+          echo "${S3CONFIG}" > $HOME/.s3cfg/config
+          chmod 600 $HOME/.s3cfg/config
+          s3cmd --config=$HOME/.s3cfg/config get s3://gadopt/github-actions/pygplates_0.36.0_py310_ubuntu-22.04.deb
+          sudo apt install -y --fix-broken ./pygplates_0.36.0_py310_ubuntu-22.04.deb
+          ln -s /usr/lib/pygplates.so /home/firedrake/firedrake/lib/python3.10/site-packages/
```

Comment:
Given Firedrake has updated to 24.04, this will have to change (as you've probably seen from the failing runs) -- I'm building a new .deb now that should work for us.

Reply:
Thanks!